dev/source/docs/plane-sitlmavproxy-tutorial.rst

```diff
@@ -51,14 +63,13 @@ Taking off
 To take off with Plane you should use a mission (AUTO mode) that
 contains the
 :ref:`MAV_CMD_NAV_TAKEOFF <plane:mav_cmd_nav_takeoff>`
-command. Once you are airborne you can switch to other other :ref:`flight modes <plane:flight-modes>`.
+command or in ``GUIDED`` mode using the takeoff console command. Once you are airborne you can switch to other other :ref:`flight modes <plane:flight-modes>`.
```

Comment:
Only works for quadplanes in guided AFAIK - it may be difficult to make this clear without getting wordy, but it has to be mentioned :-)

Reply:
np...will verify and change accordingly...